CMakeLists.txt

```diff
@@ -320,12 +320,14 @@ if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
        src/unix/linux-syscalls.c
        src/unix/procfs-exepath.c
        src/unix/random-getrandom.c
-       src/unix/random-sysctl.c
+       src/unix/random-sysctl-linux.c
        src/unix/sysinfo-loadavg.c)
 endif()
 
 if(CMAKE_SYSTEM_NAME STREQUAL "NetBSD")
-  list(APPEND uv_sources src/unix/netbsd.c)
+  list(APPEND uv_sources
+       src/unix/netbsd.c
+       src/unix/random-sysctl-bsd.c)
```

Comment:
Would it make sense to call it random-sysctl-netbsd.c for now? I understand from the discussion it's not clear whether other BSDs would use this implementation.